Resources/views/pjax_admin.html.twig

```diff
@@ -25,3 +25,19 @@
     </header>
     {% block content %}{% endblock %}
 </div>
+
+{% block notification %}
+{# Possible states are "error", "started", and "done" #}
```

Comment:
actually I would not mention the possible states as we'll probably add new ones and this list will eventually become wrong